sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -3,6 +3,7 @@
     <Description>This is the implementation of the Azure Client Pipeline</Description>
     <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
     <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
I was imagining this would simply be: `<VersionPrefix>1.0.0-preview.10</VersionPrefix>` or perhaps just `<Version>1.0.0-preview.10</Version>` if we can be sure that the we can successfully append the daily dev number to that property.

Reply:
After a quick look at https://github.com/dotnet/sdk/blob/2eb6c546931b5bcb92cd3128b93932a980553ea1/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DefaultAssemblyInfo.targets I think we can use just the `<Version>`.

Reply:
OK - so if I ammend my PRs for each pipeline to add in the explicit version numbers into the packages I am enabling unified pipelines for we should be golden.

Reply:
Great thanks. I will work with @chidozieononiwu to make sure we can still correctly produce the dev versions after this change.

Reply:
OK then - I think this one is officially ready to go then!